civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -53,6 +53,25 @@
   }
 }
 
+.ui-select-multiple {
+  padding: 3px !important;
+
+  &:after {
+    content: '\f0d7';
+    bottom: 0;
+    font-family: "FontAwesome";
+    display: inline-block;
+    height: 33px;
+    line-height: 35px;
+    position: absolute;
+    right: 0;
+    text-align: center;
+    width: 32px;
+    z-index: 1;
+    -webkit-font-smoothing: antialiased;
```

Comment:
font smoothing has limited support [across browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth)? Also only webkit is used here?